pkg/generators/golang/clients_generator.go

```diff
@@ -860,16 +869,22 @@ func (g *ClientsGenerator) generateRequestSource(method *concepts.Method) {
 			result = &{{ $responseName }}{}
 			result.status = response.StatusCode
 			result.header = response.Header
+			reader := bufio.NewReader(response.Body)
+			_, err = reader.Peek(1)
+			if err == io.EOF {
+				err = nil
```

Comment:
@jhernand why was this err set to nil here, but not in line 380 (https://github.com/openshift-online/ocm-api-metamodel/pull/162/files#diff-383dca8b1c11e0370ea35f8b4e29fe1c5a3b42220821d8db800fbf34c6b48975R380)?

Wonder if it's on purpose or just short sight. Asking because of [OCM-17408](https://issues.redhat.com/browse/OCM-17408)